src/uct/api/uct.h

```diff
@@ -843,6 +843,17 @@ enum uct_md_mem_flags {
      */
     UCT_MD_MEM_GVA                  = UCS_BIT(11),
 
+    /**
+     * Register a memory window: a shallow copy of some existing UCT memory
+     * handle, which can be used to access the same memory region. When created,
+     * the memory window inherits the original memh access flags and state, and
+     * takes ownership of the indirect keys of the original memory handle. The
```

Comment:
indirect key is IB specific we should not mention it in UCT API

Reply:
ok